CONTRIBUTING.md

```diff
@@ -57,97 +52,14 @@ Collaborators sign-off on a pull request by explicitly stating their approval in
 
 Exception to this process is allowed for high-priority changes that address security vulnerabilities. A shorter review period and modified sign-off process may be necessary depending on the nature of the change and severity of the issue. 
 
-## Release quality
-The guiding light, at all times, should be the long-term stability, reliability, maintainability, extensability, performance, and ultimately QUALITY as informed through current industry standards and best practices.
+## Release policy
+The guiding light, at all times, should be the long-term stability, reliability, maintainability, extensibility, performance, and ultimately quality as informed through current industry standards and best practices.
 
-* All pull requests should be of production quality, stable, and ready for deployment. 
-* Every PR should include updates to release notes, documentation, code comments, and incrementing the current version number.
-* After a PR has merged into master a new production build is automatically deployed.
+* Pull requests should be of production quality, stable, and ready for deployment. 
+* Pull requests should include updates to release notes, documentation, code comments, and incrementing the current version number.
```

Comment:
"...**incrementing** the current version number" - If _necessary,_ correct? If so, I would write: "...incrementing the current version number (if necessary)."